ulid/__init__.py

```diff
@@ -1,11 +1,10 @@
 from __future__ import annotations
 
+import datetime
 import functools
 import os
 import time
 import uuid
-from datetime import datetime
-from datetime import timezone
```

Comment:
It is much nicer to write and read code that does `from datetime import datetime, timezone`

Reply:
Is this really necessary in order to get rid of the Sphinx warnings?

Reply:
There must be other ways!